CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing
+
+Thank you for your interest in contributing! We welcome all contributions no matter their size. Please read along to learn how to get started. More information can be found in each of the README.md as well as  https://docs.airswap.io/ about the individual repos.
+
+## Set up:
+First fork airswap-protocols repository and then clone the project. 
+
+Example:
+
+`git clone https://github.com/airswap/airswap-protocols`
+
+This project uses lerna to handle managing the mono-repo. Lerna is a tool for managing JavaScript projects with multiple packages.
+
+Run the below command from the root directory to start downloading necessary packages:
+
+`yarn install `
+
+
+## Running the tests
+A great way to explore the code base is to run the tests.
+
+We can run all tests with:
+
+`yarn test`
+
+
+## Code Style
+When multiple people are working on the same body of code, it is important that conforms to a similar style. It often doesn’t matter as much which style, but rather that they conform to one style.
+
+To ensure your contribution conforms to the style being used in this project, we require that a linter is run prior to committing to ensure styling.
+
+Run:
+
+```
+yarn lint (linting for the Javascript)
+yarn hint (linting for the Solidity code)
+```
```

Comment:
should be publish our style guide too? To show naming conventions etc?

Reply:
good call, adding it from notion page into it's own md page,